### PR TITLE
NXP-26372: Add step to validate the value of the results count label on searches

### PR DESCRIPTION
**Pull request by FabioFSantos**

### CODE REVIEW

File: packages/nuxeo-web-ui-ftest/features/step_definitions/search.js [hunk 1]
```diff
@@ -108,6 +108,13 @@ Then(/^I can see (\d+) search results$/, function(numberOfResults) {
   }
 });
 
+Then(/^I can see that the results count label value is (\d+)$/, function(labelCount) {
+  const label = this.ui.results.resultsCountLabel;
+  label.waitForVisible();
+  const count = label.getText();
+  parseInt(count, 10).should.equal(labelCount);
```
Comment (guirenard) [inline packages/nuxeo-web-ui-ftest/features/step_definitions/search.js]: Can't we imagine to simply merge this check in the `Then(/^I can see (\d+) search results$/, function(numberOfResults)` step?

Comment (nmpcunha) [reply packages/nuxeo-web-ui-ftest/features/step_definitions/search.js]: It is a fair point!

Comment (FabioFSantos) [reply packages/nuxeo-web-ui-ftest/features/step_definitions/search.js]: 👍 https://github.com/nuxeo/nuxeo-web-ui/pull/797/commits/0addc84b513c9298a0bed9003a5e04d75b8fcf1b

File: packages/nuxeo-web-ui-ftest/features/step_definitions/search.js [hunk 2]
```diff
@@ -101,6 +101,9 @@ Then(/^I can see (\d+) search results$/, function(numberOfResults) {
     this.ui.results.noResults.waitForVisible().should.be.true;
   } else {
     this.ui.results.getResults(displayMode).waitForVisible();
+    this.ui.results.resultsCountLabel.waitForVisible();
+    const count = this.ui.results.resultsCountLabel.getText();
+    parseInt(count, 10).should.equal(numberOfResults);
```
Comment (nmpcunha) [inline packages/nuxeo-web-ui-ftest/features/step_definitions/search.js]: nit: maybe joining these two lines?

`parseInt(this.ui.results.resultsCountLabel.getText(), 10).should.equal(numberOfResults);`

Comment (nmpcunha) [reply packages/nuxeo-web-ui-ftest/features/step_definitions/search.js]: Despite that, it seems good 👍

**Comment by FabioFSantos** [comment]
https://qa.nuxeo.org/jenkins/view/TestAndPush/job/web-ui-pipeline-fdsantos/68/ 🎉

**Pull request merged**